src/main/java/step1/StringCalculator.java

```diff
@@ -0,0 +1,61 @@
+package step1;
+
+import java.util.Objects;
+
+public class StringCalculator {
+
+    private final Operators operators;
+    private final Numbers numbers;
+
+    public StringCalculator() {
+        operators = new Operators();
+        numbers = new Numbers();
+    }
+
+    public StringCalculator(Operators operators, Numbers numbers) {
+        this.operators = operators;
+        this.numbers = numbers;
+    }
+
+    public void addElement(String element) {
+        try {
+            numbers.addNumber(element);
+        } catch (NumberFormatException e) {
+            operators.addOperator(element);
+        }
+    }
+
+    public int calculate() {
+        if (!hasOperatorsAndNumbers())
+            throw new RuntimeException("초기 연산자와 숫자 세팅이 이루어지지 않았습니다.");
```

Comment:
코드가 한 줄밖에 없더라도 중괄호는 추가해 주세요
```suggestion
        if (!hasOperatorsAndNumbers()) {
            throw new IllegalArgumentException("초기 연산자와 숫자 세팅이 이루어지지 않았습니다.");
        }    
```